docs/kh3/index.md

```
@@ -8,3 +8,4 @@
     * [Pawns](pawns.md)
 * Other
     * [Battle Quotes](battlequotes.md)
+    * [USM Files (CriMovie)](crimovie.md)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)